PostHog/PostHogMaskViewModifier.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  PostHogMaskViewModifier.swift
+//  PostHog
+//
+//  Created by Yiannis Josephides on 09/10/2024.
+//
+
+#if os(iOS) && canImport(SwiftUI)
+
+    import SwiftUI
+
+    public extension View {
+        func postHogMask(_ value: Bool = true) -> some View {
+            modifier(PostHogMaskViewModifier(enabled: value))
+        }
+    }
+
+    private struct PostHogMaskViewTagger: UIViewRepresentable {
+        func makeUIView(context _: Context) -> PostHogMaskViewTaggerView {
+            PostHogMaskViewTaggerView()
+        }
+
+        func updateUIView(_: PostHogMaskViewTaggerView, context _: Context) {
+            // nothing
+        }
+    }
+
+    private struct PostHogMaskViewModifier: ViewModifier {
+        let enabled: Bool
+        @State private var visible = false
+
+        func body(content: Content) -> some View {
+            content.background(viewTagger)
+        }
+
+        @ViewBuilder
+        private var viewTagger: some View {
+            if enabled {
+                PostHogMaskViewTagger()
+            }
+        }
+    }
+
+    private class PostHogMaskViewTaggerView: UIView {
+        override func didMoveToSuperview() {
```

Comment:
is there a method to override that we would set `phIsManuallyMasked` to false? so maybe fixes the issues with the open sheet